source/Mission.cpp

```diff
@@ -703,6 +703,20 @@ bool Mission::CanOffer(const PlayerInfo &player, const shared_ptr<Ship> &boardin
 
 
 
+bool Mission::CanAccept(const PlayerInfo &player) const
+{
+	auto it = actions.find(OFFER);
```

Comment:
This code should be fine for jobs, but should it also be tested for other (non-jobs) types of missions? (Does the same issue as that is solved for jobs here also happen for other types of missions)?